src/test/java/com/google/api/codegen/testdata/nodejs_doc_message_library.baseline

```diff
@@ -756,21 +756,69 @@ var Book = {
  * @property {Object} field2
  *   This object should have the same structure as [SomeMessage2]{@link SomeMessage2}
  *
+ * @property {number} alignment
+ *   The number should be among the values of [Alignment]{@link Alignment}
```

Comment:
@jmuk for context, this PR is connected to a bug in Python where there are multiple enums of the same simple name in an API. (For example, the Vision API has two different enums called `Type`.) Does NodeJS handle this properly currently, i.e., between `[Alignment]{@link Alignment}` on this line, and on line 792, which represent two different enums?